recipes-extended/sigslot/sigslot_1.2.1.bb

```
@@ -11,3 +11,6 @@ SRCREV = "e41c6e62edfa67b6402b17a627dca698a14f47a8"
 S = "${WORKDIR}/git"
 
 inherit cmake
+
+# header only library
+ALLOW_EMPTY:${PN} = "1"
```

I see this used in other recipes, but according to this comment:
https://docs.yoctoproject.org/pipermail/yocto/2014-May/019789.html
and the number of recipes in meta-oe I see using the suggestion, it seems this:
RDEPENDS:${PN}-dev = ""
is the recommended solution.
(Untested.)
```suggestion
RDEPENDS:${PN}-dev = ""
```